README.md

```diff
@@ -1,6 +1,5 @@
 # A Harvest Java Client
 
-[![CircleCI](https://circleci.com/gh/3AP-AG/harvest-client.svg?style=svg)](https://circleci.com/gh/3AP-AG/harvest-client)
```

Comment:
Let's have a GHA badge here https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge of the latest `develop` branch

Reply:
Moved the `pull-request` badge to the first place.